internal/serve/serve.go

```diff
@@ -4,22 +4,31 @@ import (
 	"fmt"
 	"net/http"
 
+	"wallet-backend/internal/data"
+	"wallet-backend/internal/db"
+
+	"github.com/pkg/errors"
 	supporthttp "github.com/stellar/go/support/http"
 	supportlog "github.com/stellar/go/support/log"
 	"github.com/stellar/go/support/render/health"
 )
 
 type Configs struct {
-	Logger *supportlog.Entry
-	Port   int
+	Logger          *supportlog.Entry
+	Port            int
+	DBConnectionURL string
```

Comment:
Can we simply name this DatabaseURL? I don't see where we read the env and assign the value to this variable. Should we add it to cmd/serve.go?

Reply:
Added: [fc7fee0](https://github.com/stellar/wallet-backend/pull/5/commits/fc7fee097ada767a932d5ec638cad99d78a3ce49)